.github/workflows/continuous-integration-javascript.yml

```diff
@@ -4,7 +4,6 @@ on: [push]
 
 env:
   CI: true
-  FORCE_COLOR: '1' # https://github.com/chalk/chalk/issues/203
```

Comment:
I think we should clean up `src/sx-tailwind-website/__github__/.github/workflows/continuous-integration.yml` as well.